.github/workflows/code-rabbit-scan.yml

```
@@ -0,0 +1,28 @@
+  pull_request:
+    branches:
+      - main
+      - changes  # Adjust this branch name to your desired branches
+
+jobs:
+  scan:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
```

actionlint

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)